app/src/index.css

```diff
@@ -7,7 +7,14 @@
 body {
   margin: 0;
   padding: 0;
-  background: radial-gradient(#051223, #051223, #030e1c);
+  background: radial-gradient(
```

Comment:
**Problem:** Weird background artifact and color transition
**Proposal:** Make the color shift more progressive in order to match the image better and adjust radius to remove center artifact.
![image](https://user-images.githubusercontent.com/64640406/181270888-a977902d-475a-4ca0-8eac-6e1027ed4ed1.png)

This tool might be useful: [CSS Gradient](https://cssgradient.io/)

Reply:
As discussed will try a different approach with seperate background and animation